modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -1174,7 +1179,26 @@ public boolean isSatisfiedBy(Task task) {
 
 				@Override
 				public void execute(Project p) {
-					verify.checksum(workspaceExtension.getBundleChecksumMD5());
+					if (!Objects.equals(
+							workspaceExtension.getBundleUrl(),
+							workspaceExtension.getDefaultBundleUrl()) &&
+						Objects.isNull(_userBundleCheckSumMD5)) {
+
+						verify.setEnabled(false);
```

Comment:
@simonjhy  you should use `onlyIf` spec closure instead.

Reply:
ok,got it.